collectors/python.d.plugin/rabbitmq/rabbitmq.chart.py

```diff
@@ -104,6 +114,28 @@
             ['object_totals_exchanges', 'exchanges', 'absolute']
         ]
     },
+    'connection_churn_rates': {
+        'options': [None, 'Connection Churn Rates', 'operations/s', 'overview', 'rabbitmq.connection_churn_rates', 'line'],
+        'lines': [
+            ['churn_rates_connection_created_details_rate', 'connections opened', 'absolute'],
+            ['churn_rates_connection_closed_details_rate', 'connections closed', 'absolute']
```

Comment:
Since we have 
- connection/channel/queue in the title
- we split them into separate charts
- units is operations/s

i think we can simplify dimension names:

- connections opened => open (read as open operations/s )
- connections closed => close (read as close operations/s)

Does it looks ok for you?

Reply:
Nice idea.  There is not much space in the legends section.